### PR TITLE
Feature/anshul birla/1318341695 multi lang upload

### PR DESCRIPTION
**Pull request by Anshul-Birla**
### Administrative Info
Monday Board ID: 853642866
Make sure your branch name conforms to: `<feature/staging/hotfix/...>/[username]/[Monday Item ID]-[3-4 word description separated by dashes]`. Otherwise, please rename your branch and create a new PR.

### Changes
What changes did you make?
- Added ability to upload books in multiple languages 

### Testing
How did you confirm your changes worked? 
- Tested uploading books in all, none, and varying amounts of languages. Went through wizard multiple times

### Confirmation of Change 
Go to upload books page, navigate to the new button, and start uploading the books. Look for drop down on page 2 to see the changes

### CODE REVIEW

File: web/src/components/UploadBooksDropdown.module.css [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+.firstDropdownElement {
+    composes: dropdownElement;
+    border-radius: 10px 10px 0 0;
+}
+
+.lastDropdownElement {
+    composes: dropdownElement;
+    border-radius: 0 0 10px 10px;
+}
```
Comment (dacruz21) [inline web/src/components/UploadBooksDropdown.module.css:30]: This is web so we actually have fully featured CSS!!! Can you use :first/last-of-type to combine these and get rid of the annoying index logic?

https://www.w3schools.com/cssref/sel_first-of-type.asp
https://www.w3schools.com/cssref/sel_last-of-type.asp

File: web/src/pages/BookWizard/TabContentPage.tsx [hunk 2]
```diff
@@ -34,6 +36,11 @@ export const TabContentPage: React.FC<TabConentPageProps> = ( {onContentChange,
     setVideo(currentContent.video);
   }, []);
 
+  useEffect( () => {
+    setBody(currentContent.body);
+    setVideo(currentContent.video);
+  }, [language]);
```
Comment (dacruz21) [inline web/src/pages/BookWizard/TabContentPage.tsx:42]: I think currentContent would be the better dependency here right? Language probably works too but currentContent should work and makes more sense.

**Review by typokign — CHANGES_REQUESTED** [review]
Couple small things but looks good!

File: web/src/pages/BookWizard/TabContentPage.tsx (hunk 2 above)
Comment (Anshul-Birla) [reply web/src/pages/BookWizard/TabContentPage.tsx:42]: if you do current content, react complains that the "maximum depth has been exceeded". current content change causes video and body to be changed,which caused the reac tab content map to be changed (in the parent), which updates the current content again. It loop infinitely

**Review by typokign — APPROVED** [review]
LGTM, sorry for the late review :)

**Pull request merged**